__tests__/ncnewsapp.test.js

```diff
@@ -81,5 +81,26 @@ describe('/api/articles/:article_id', () => {
   });
 });
 
+describe('/api/articles', () => {
+  test('GET:200 Responds with an articles array of article objects, each of which with specific properties: ', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles[0]).toEqual( 
+          {
+            article_id: 3,
+            title: 'Eight pug gifs that remind me of mitch',
+            topic: 'mitch',
+            author: 'icellusedkars',
+            created_at: "2020-11-03T09:12:00.000Z",
+            votes: 0,
+            comment_count: '2'
```

Comment:
Either within this test, or a new one, we need to be checking that the response has come back in date order, descending 👍